runtime/memory/64bit_nix.c

```diff
@@ -50,9 +50,13 @@ i32 instruction_memory_grow(i32 count) {
     return prev_size;
 }
 
-INLINE char* get_memory_ptr_for_runtime(u32 offset, u32 bounds_check) {
+INLINE void check_bounds(u32 offset, u32 bounds_check) {
     // Due to how we setup memory for x86, the virtual memory mechanism will catch the error, if bounds < WASM_PAGE_SIZE
     assert(bounds_check < WASM_PAGE_SIZE || (memory_size > bounds_check && offset <= memory_size - bounds_check));
```

Comment:
OT: this is probably not the most efficient assert here. This may be more now that you're using this code more. Can be deferred to the future

Reply:
Opened as #49